debezium-event-api/pom.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
```

Comment:
the more Maven modules we add, the less likely we'll ever have separate GitHub repos for the different components (an idea which I started to like :) )